services/QuillLMS/spec/factories/student_response_answer_text.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :student_response_answer_text, class: 'StudentResponseAnswerText' do
+    sequence(:text) { |n| "This a student response answer #{n}." }
```

Comment:
I think the naming of this field even tricked yourself (you have this as a `jsonb` column). Again, let's change that name.

Reply:
Name updated.